pkg/tracing/context.go

```diff
@@ -0,0 +1,28 @@
+package tracing
+
+import (
+	"context"
+
+	"github.com/influxdata/influx-cli/v2/api"
+)
+
+// Context is a convenience wrapper for tracking a global tracing
+// ID through CLI commands.
+type Context interface {
```

Comment:
This was my thought for how to deal with the global trace ID parameter without adding an extra arg to every function in the program; open to other suggestions.